### PR TITLE
Friendly pirate hails

### PR DESCRIPTION
**Pull request by Amazinite**
I felt lonely when I was friendly with pirates. No one would talk to me. Now they do!

Decided to not bundle this with the pirate jobs because they work independently of each other.
The first two friendly hails in the set are just slightly reworded hostile hails.

### CODE REVIEW

**Comment by Pointedstick** [comment]
Excellent! I like the drug references. That helps establish what those things are.

File: data/hails.txt [hunk 1]
```diff
@@ -1494,7 +1494,333 @@ phrase "friendly pug"
...
+phrase "friendly pirate"
+	word
+		"Anyone"
+		"Anyone here"
+		"Any fellow pirates"
+		"Any fellow anarchists"
+	word
+		" "
+	word
+		"up to buying"
```
Comment (endless-sky) [inline data/hails.txt]: I think this should be "up for" instead of "up to."

**Comment by Amazinite** [comment]
Changed it. :+1:

**Comment by Amazinite** [comment]
:firework:

File: data/hails.txt [hunk 2]
```diff
@@ -1494,7 +1494,333 @@ phrase "friendly pug"
...
+		"throwing them out the airlock"
+		"starving them until they listen"
+		"threatening to throw them out the airlock"
+		"threatening to kill them"
+		"threatening to shoot them"
+	word
+		"."
+	word
+		" It worked for me!"
+		" I guarentee it works!"
```
Comment (Bladewood) [inline data/hails.txt]: guarantee*

**Pull request merged**